lib/DDG/Spice/Metar.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Metar;
+
+# Metar: Looks up the METAR information for a given airport. 
+# The input should be a 4 letter ICAO code.
+# It recognizes the query "metar ICAO" and "ICAO metar". 
+
+use DDG::Spice;
+
+spice to => 'http://avwx.rest/api/metar.php?station=$1&format=JSON';
+spice proxy_cache_valid => "418 1d"; # Do not cache responses.
+spice wrap_jsonp_callback => 1;
+
+triggers any => 'metar';
+
+handle words => sub {
```

Comment:
> I implemented it that way because the docs said to use trigger words and avoid regexes whenever possible.

To clarify I meant that we'd still use trigger words, but there's no harm in using a regex inside the `handle` to verify the query is relevant.

Using a regex **_trigger_** is inefficient because it means it must be run against every query. With trigger words we just check if a query contains a particular word and only then would our `handle` execute meaning we'd only run the regex when we have a strong indicator the query may be relevant to our IA :+1: